smslibrary/src/test/java/com/eis/smslibrary/SMSManagerTest.java

```diff
@@ -53,50 +45,12 @@ public void singletonInstance() {
 
     @Test()
     public void validMessage_isSent() {
-        ArrayList<String> messageText = new ArrayList<>(1);
-        messageText.add(VALID_MESSAGE_TEXT);
-
-        SmsManager managerMock = mock(SmsManager.class);
-        when(managerMock.divideMessage(anyString())).thenReturn(messageText);
-
-        PowerMockito.mockStatic(SmsManager.class);
-        when(SmsManager.getDefault()).thenReturn(managerMock);
-
-        managerInstance.sendMessage(VALID_MESSAGE);
-
-        verify(managerMock).sendMultipartTextMessage(peerNumberCaptor.capture(),
-                isNull(String.class), messageTextCaptor.capture(), isNull(ArrayList.class),
-                isNull(ArrayList.class));
-        assertEquals(peerNumberCaptor.getValue(), VALID_PEER_NUMBER);
-        assertEquals(messageTextCaptor.getValue(), messageText);
```

Comment:
Why did you remove all these lines of tests? I think the reason you have non-working tests is not because of #20 (they were all working there), but because something went wrong when you merged [`master` from the main repository](/Cogno-IDU/smslibrary/tree/master) into [your branch](/EIS0/smslibrary/tree/feature/powermock-tests).